packages/core/src/sync-store/sqlite/store.ts

```diff
@@ -434,30 +485,37 @@ export class SqliteSyncStore implements SyncStore {
                       f.topic0,
                     )}, ${sql.val(f.topic1)}, ${sql.val(f.topic2)}, ${sql.val(
                       f.topic3,
-                    )} )`,
+                    )}, ${sql.lit(f.includeTransactionReceipts)} )`,
                 ),
               )} )`,
           )
           .selectFrom("factoryLogFilterIntervals")
-          .leftJoin("factories", "factoryId", "factories.id")
+          .innerJoin("factories", "factoryId", "factories.id")
           .innerJoin("factoryFilterFragments", (join) => {
-            let baseJoin = join.on(({ and, cmpr }) =>
-              and([
-                cmpr("fragmentAddress", "=", sql.ref("address")),
-                cmpr("fragmentEventSelector", "=", sql.ref("eventSelector")),
-                cmpr(
+            let baseJoin = join.on((eb) =>
+              eb.and([
+                eb("fragmentAddress", "=", sql.ref("address")),
+                eb("fragmentEventSelector", "=", sql.ref("eventSelector")),
+                eb(
                   "fragmentChildAddressLocation",
-                  "=",
+                  "<=",
```

Comment:
This seems like a merge conflict or something that I did, should be "<=" for the transactionReceipts boolean